cibyl/sources/elastic/filter.py

```diff
@@ -0,0 +1,142 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+import argparse
+import logging
+from cibyl.sources.elastic.client import ElasticSearchClient
```

Comment:
same as above, separate imports by: 3rd party libraries, project modules and python standard library modules

Reply:
I can see it now in the`imports` side: https://www.python.org/dev/peps/pep-0008/#imports

Thx! Change done.